build.gradle

```diff
@@ -651,6 +651,28 @@ task testDocker {
   }
 }
 
+task acceptanceTestsGoQuorum {
+  dependsOn distDocker
+
+  def tags = "(basic || basic-istanbul || networks/typical-besu::ibft2)"
+
+  doLast {
+    // TODO ricardolyn: do we need multiple variants for GoQ AT?
```

Comment:
@macfarla @pinges should we run the GoQ AT against only one variant or all variants?

Reply:
other variants of jdk? I think one is enough

Reply:
yes, I meant JDK. ok, then I will leave only one and remove the TODO. thank you